src/SDKs/Storage/Management.Storage/Generated/Models/CustomDomain.cs

```diff
@@ -33,13 +33,13 @@ public CustomDomain()
         /// </summary>
         /// <param name="name">Gets or sets the custom domain name assigned to
         /// the storage account. Name is the CNAME source.</param>
-        /// <param name="useSubDomain">Indicates whether indirect CName
+        /// <param name="useSubDomainName">Indicates whether indirect CName
         /// validation is enabled. Default value is false. This should only be
         /// set on updates.</param>
-        public CustomDomain(string name, bool? useSubDomain = default(bool?))
+        public CustomDomain(string name, bool? useSubDomainName = default(bool?))
         {
             Name = name;
-            UseSubDomain = useSubDomain;
+            UseSubDomainName = useSubDomainName;
```

Comment:
Please note that this is potentially a breaking change

Reply:
Yes, this breaking is intend. Since the before "UseSubDomain" is wrong. It should be "UseSubDomainName".